test/suites/standard2/gateway-api.robot

```diff
@@ -0,0 +1,82 @@
+*** Settings ***
+Documentation       Test Gateway API functionality
+
+Resource            ../../resources/microshift-network.resource
+
+Suite Setup         Setup Suite With Namespace
+Suite Teardown      Teardown Suite With Namespace
+
+
+*** Variables ***
+${NS_GATEWAY_1}                 ${EMPTY}
+${GATEWAY_MANIFEST_TMPL}        ./assets/gateway-api/gateway.yaml.template
+${HTTP_ROUTE_MANIFEST_TMPL}     ./assets/gateway-api/http-route.yaml.template
+${GATEWAY_1_HOSTNAME}           gw1.microshift.local
+${GATEWAY_1_PORT}               9000
+
+
+*** Test Cases ***
+Test Simple HTTP Route
+    [Documentation]    Create a gateway and test it with Hello MicroShift application with HTTPRoute.
+    [Setup]    Run Keywords
+    ...    Setup Namespaces
+    ...    Deploy Hello MicroShift
+    Create Gateway    hostname=${GATEWAY_1_HOSTNAME}    port=${GATEWAY_1_PORT}    namespace=${NS_GATEWAY_1}
+    Create HTTP Route    hostname=${GATEWAY_1_HOSTNAME}    namespace=${NS_GATEWAY_1}
+    Access Hello MicroShift Success    ushift_port=${GATEWAY_1_PORT}    hostname=${GATEWAY_1_HOSTNAME}
+    [Teardown]    Run Keywords
+    ...    Delete Namespaces
+
+
+*** Keywords ***
+Deploy Hello MicroShift
+    [Documentation]    Deploys the hello microshift application (service included)
+    ...    in the given namespace.
+    Create Hello MicroShift Pod    ns=${NS_GATEWAY_1}
+    Expose Hello MicroShift    ${NS_GATEWAY_1}
+
+Setup Namespaces
+    [Documentation]    Configure the required namespaces for namespace ownership tests.
+    Set Suite Variable    \${NS_GATEWAY_1}    ${NAMESPACE}-gw-1
+    Create Namespace    ${NS_GATEWAY_1}
```

Comment:
Documentation says "configure ... namespaces" but the keyword only creates a new namespace. Do we need the extra namespace?